docs/reference/parts.rst

```diff
@@ -0,0 +1,14 @@
+.. _parts:
```

Comment:
I ran into an oddity. The first link in [Filesets](https://canonical-charmcraft--1939.com.readthedocs.build/en/1939/common/craft-parts/explanation/filesets/) works but there's a missing second link that should be pointing to `lifecycle.rst` in Craft Parts. I believe the first link works because the reference is colliding with reference on this line here.

The missing link was causing a local build error for me. By adding `.. _lifecycle:` to a random file, the build succeeded.

Reply:
It wasn't erroring out for me, but I've added some brief lifecycle docs that mostly just call out the craft-parts docs.